src/OpenSpaceToolkit/Astrodynamics/EventCondition/LogicalConnective.cpp

```diff
@@ -0,0 +1,42 @@
+/// Apache License 2.0
+
+#include <OpenSpaceToolkit/Astrodynamics/EventCondition/LogicalConnective.hpp>
+
+namespace ostk
+{
+namespace astro
+{
+namespace eventcondition
+{
+LogicalConnective::LogicalConnective(const String& aName, const Array<Shared<EventCondition>>& eventConditions)
+    : EventCondition(aName, EventCondition::Criteria::Undefined),
+      eventConditions_(eventConditions)
+{
+}
+
+LogicalConnective::~LogicalConnective() {}
+
+Array<Shared<EventCondition>> LogicalConnective::getEventConditions() const
```

Comment:
If this is called a lot, might make sense to add an `accessEventConditions` and use that instead.

Reply:
Fair point. Currently it's not used at all, I feel like it will mostly just be used in python. But I'll keep that in mind and add it in the future if we required it.